X/xxHash/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "xxHash"
+version = v"0.7.3"
+
+# Collection of sources required to complete build
+sources = [
+    FileSource("https://github.com/Cyan4973/xxHash/archive/v0.7.3.tar.gz", "952ebbf5b11fbf59ae5d760a562d1e9112278f244340ad7714e8556cbe54f7f7"),
```

Comment:
We recently renamed this type:
```suggestion
    ArchiveSource("https://github.com/Cyan4973/xxHash/archive/v0.7.3.tar.gz", "952ebbf5b11fbf59ae5d760a562d1e9112278f244340ad7714e8556cbe54f7f7"),
```
:wink: